ETL/test/handle.cpp

```diff
@@ -327,6 +327,24 @@ int rhandle_general_use_test(void)
 		printf(__FILE__":%d: On clear, instance count=%d, should be zero.\n",__LINE__,my_test_obj::instance_count);
 		return 1;
 	}
+    obj_handle obj1;
```

Comment:
Please add a description for the tests. 
It is also better to add how many references should be in `etl::rhandle` and `etl::handle` after copy and move operations.